bin/fixmystreet.com/call-wasteworks-backend

```diff
@@ -17,22 +17,26 @@ use Getopt::Long::Descriptive;
 use JSON::MaybeXS;
 use Storable;
 use Integrations::Echo;
+use Integrations::Bartec;
 use FixMyStreet::Cobrand;
 
 my ($opts, $usage) = describe_options(
     '%c %o',
     ['cobrand=s', 'which cobrand configuration to use'],
+    ['backend=s', 'which backend type (echo/bartec)'],
     ['out=s', 'where to output data'],
     ['calls=s', 'JSON of what API calls to make'],
     ['help|h', "print usage message and exit" ],
 );
 $usage->die if $opts->help;
-$usage->die unless $opts->cobrand && $opts->out && $opts->calls;
+$usage->die unless $opts->cobrand && $opts->out && $opts->calls && $opts->backend;
```

Comment:
Have just pushed a change I kept meaning to PR and not getting round to, to use this at the command line more easily, good for debugging something.

Reply:
Nice, change looks good to me.